api_bugzilla.py

```diff
@@ -225,13 +267,68 @@ def clean_table(self, table):
         self.session.query(table).delete()
         self.session.commit()
 
+    def sync_bugzilla_desktop_bugs(self, payload):
```

Comment:
Rename to `update_bugzilla_desktop_bugs` or `update_bugzilla_desktop_bugs_insert`